src/browser-extensions/URLBrowserResolver.js

```diff
@@ -46,13 +49,20 @@ class URLBrowserResolver {
 		this.resolving = {};
 	}
 
-	resolve(url) {
+	resolve(unresolvedFile) {
+		let url = unresolvedFile;
+		let headers = {};
+		if (unresolvedFile && typeof unresolvedFile === 'object') {
+			url = unresolvedFile.url;
+			headers = unresolvedFile.headers || {};
+		}
+
 		if (!this.resolving[url]) {
 			this.resolving[url] = new Promise((resolve, reject) => {
-				if (url.toLowerCase().indexOf('https://') === 0 || url.toLowerCase().indexOf('http://') === 0) {
```

Comment:
This library is also used in nodejs (https://www.npmjs.com/package/node-fetch#fetchurl-options) and as you can see there, it needs an absolute URL as parameter.